site/en/docs/privacy-sandbox/attribution-reporting-experiment/index.md

```diff
@@ -2,39 +2,33 @@
 layout: 'layouts/doc-post.njk'
 title: 'Attribution Reporting: experiment and participate'
 date: 2022-03-31
-updated: 2022-12-15
+updated: 2023-09-18
 authors:
   - maudn
   - alexandrawhite
 ---
 
+{% Partial 'privacy-sandbox/ot-end.njk' %}
+
 Read [Attribution Reporting](/docs/privacy-sandbox/attribution-reporting/) to
 see the use cases and key concepts for this API and check the most recent [updates](/docs/privacy-sandbox/attribution-reporting-updates/).
 
-Learn why we plan to [ship the Attribution Reporting API](/docs/privacy-sandbox/attribution-reporting/chrome-shipping)
-in the first half of 2023.
+Learn why [shipped the Attribution Reporting API](/docs/privacy-sandbox/attribution-reporting/chrome-shipping).
```

Comment:
```suggestion
Learn why [we shipped the Attribution Reporting API](/docs/privacy-sandbox/attribution-reporting/chrome-shipping).
```

Reply:
Is that what it correctly describes? Or is it specific to the timeline?

Reply:
thank you! yes fixing